pkg/kubelet/nodelease/controller_test.go

```diff
@@ -195,3 +200,84 @@ func TestNewLease(t *testing.T) {
 		})
 	}
 }
+
+func TestRetryUpdateLease(t *testing.T) {
+	node := &corev1.Node{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "foo",
+			UID:  types.UID("foo-uid"),
+		},
+	}
+	noConnectionUpdateErr := apierrors.NewServerTimeout(schema.GroupResource{Group: "v1", Resource: "lease"}, "put", 1)
+	optimistcLockUpdateErr := apierrors.NewConflict(schema.GroupResource{Group: "v1", Resource: "lease"}, "lease", fmt.Errorf("conflict"))
+	cases := []struct {
+		desc                       string
+		updateReactor              func(action clienttesting.Action) (handled bool, ret runtime.Object, err error)
```

Comment:
remove names of parameters in the here and below

Reply:
Done.